server/php-worker-stats.h

```diff
@@ -57,7 +64,24 @@ class PhpWorkerStats {
   std::array<int64_t, PERCENTILE_SAMPLES> script_memory_used_samples_{};
   std::array<int64_t, PERCENTILE_SAMPLES> script_real_memory_used_samples_{};
 
-  size_t circular_percentiles_counter_{0};
+  size_t worker_circular_percentiles_counter_{0};
+  std::array<int64_t, PERCENTILE_SAMPLES> workers_script_heap_usage_bytes_{};
+  std::array<uint32_t, PERCENTILE_SAMPLES> workers_malloc_non_mapped_total_used_bytes_{};
+  std::array<uint32_t, PERCENTILE_SAMPLES> workers_malloc_non_mapped_allocated_bytes_{};
+  std::array<uint32_t, PERCENTILE_SAMPLES> workers_malloc_non_mapped_free_bytes_{};
+  std::array<uint32_t, PERCENTILE_SAMPLES> workers_malloc_mmaped_bytes_{};
+
+  std::array<int64_t, PERCENTILE_SAMPLES> workers_rss_usage_{};
+  std::array<int64_t, PERCENTILE_SAMPLES> workers_vms_usage_{};
+  std::array<int64_t, PERCENTILE_SAMPLES> workers_shm_usage_{};
+
+  uint64_t rss_kb_sum_{0};
+  uint64_t vms_kb_sum_{0};
+
+  uint32_t rss_kb_max_{0};
+  uint32_t vms_kb_max_{0};
```

Comment:
Why do we use `uint64_t` for `rss_kb_sum_` but `uint32_t` for `rss_kb_max`?

Reply:
Yeah..

rss_kb_max - in kbytes, so uint32_t is enough here
rss_kb_sum_ - sum for all processes in kbytes, and I'm not sure if uint32_t is enough for sum, but uint64_t is enough
workers_rss_usage_ - in bytes..

I'm changing rss_kb_sum_ to int64, to be more consistent.